# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/l10n/L10nOverlays.cpp
# Commit: 3404fd6725f1
# Full Hash: 3404fd6725f15d1253a7f76e3937185bd353f263
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-02 21:43:56
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part2: Switch Localization class to use localization-ffi. r=emilio,nika
#   
#   Depends on D104788
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D104789
# ==============================================================================

diff -r c9cfbba15d90 -r 3404fd6725f1 dom/l10n/L10nOverlays.cpp
--- a/dom/l10n/L10nOverlays.cpp	Mon Aug 02 09:54:07 2021 +0000
+++ b/dom/l10n/L10nOverlays.cpp	Mon Aug 02 09:54:07 2021 +0000
@@ -516,7 +516,7 @@
               DocumentFragment(aElement.OwnerDoc()->NodeInfoManager());
       nsContentUtils::ParseFragmentHTML(
           NS_ConvertUTF8toUTF16(aTranslation.mValue), fragment,
-          nsGkAtoms::_template, kNameSpaceID_XHTML, false, true);
+          nsGkAtoms::_template, kNameSpaceID_XHTML, false, true, 0);
       if (NS_WARN_IF(aRv.Failed())) {
         return;
       }